Source/WebCore/page/scrolling/ScrollSnapOffsetsInfo.cpp

```diff
@@ -324,6 +324,7 @@ void updateSnapOffsetsForScrollableArea(ScrollableArea& scrollableArea, const Re
     HashMap<LayoutUnit, SnapOffset<LayoutUnit>> verticalSnapOffsetsMap;
     HashMap<LayoutUnit, SnapOffset<LayoutUnit>> horizontalSnapOffsetsMap;
     Vector<LayoutRect> snapAreas;
+    Vector<uint64_t> snapAreasIDs;
```

Comment:
Using bare `uint64_t` is discouraged; use an `ObjectIdentifier<>` instead.

Reply:
We use the ElementIdentifier alias.